clusterloader2/cmd/clusterloader.go

```diff
@@ -31,6 +33,7 @@ import (
 
 var (
 	clusterLoaderConfig config.ClusterLoaderConfig
+	testConfigPaths     []string
```

Comment:
Why are you adding new field for this? Can't you just make the one inside clusterLoaderConfig an array?

Reply:
The problem is that if you are running test, it is hard to decide which test path from testConfigPaths you should use. Best approach for me is to have only one test config path, cause only one test config path is related to this test at a time.
That's why I decided have list of test to run outside the cluster config and provide test config path one by one.

Reply:
Ok. Wdyt about changing `ClusterLoaderConfig.TestConfigPath` -> `ClusterLoaderConfig.TestConfig` and passing the config itself directly. Will also make `RunTest()` signature simpler.

Reply:
I still need test config path to be passed to template provider - it uses relative path for object descriptions.